tests/test_functons.py

```diff
@@ -12,56 +12,88 @@
 
 
 @pytest.mark.parametrize("test_square_input,test_square_expected", [
+    (-(3037000499**2-1), -sys.maxsize-1),
+    (-(3037000499**2), -sys.maxsize),
     (-10, 100),
     (-1, 1),
     (0, 0),
     (1, 1),
-    (2, 4),
     (10, 100),
+    (3037000499**2-1, sys.maxsize-1),
+    (3037000499**2, sys.maxsize)
 ])
 def test_square(test_square_input, test_square_expected):
     assert square(test_square_input) == test_square_expected
 
 
 @pytest.mark.parametrize("test_input,expected", [
-    (-100, -99),
+    (-sys.maxsize,  -sys.maxsize+1),
+    (-sys.maxsize-1,  -sys.maxsize),
```

Comment:
the boundary here is `-sys.maxsize-1` so need (x-1, x, x+1), you only got 2 inputs for this boundary, missing `x-1` which will be `-sys.maxsize-2`